cmd/anubis/main.go

```diff
@@ -215,6 +215,7 @@ func main() {
 	h = s
 	h = internal.DefaultXRealIP(*debugXRealIPDefault, h)
 	h = internal.XForwardedForToXRealIP(h)
+	h = internal.NoStoreCache(h)
```

Comment:
I think you want to set this on the render challenge page function in lib/anubis.go

Reply:
👍🏼 Yep, I didn't consider that we probably want to cache the assets/scripts

(super unrelated, but I'm stuck on the "making sure you're not a bot..." on https://anubis.techaro.lol/docs/admin/installation/#key-generation right now...)

Reply:
I'm also stuck as another data point :sweat: (inf redirect)

Reply:
> I'm also stuck as another data point 😓 (inf redirect)

😨 It's interesting that it happens locally & on anubis.techaro.lol but not on your blog! https://jaredallard.dev/fighting-back-against-ai-in-kubernetes-with-anubis/ works fine for me

Reply:
> 😨 It's interesting that it happens locally & on anubis.techaro.lol but not on your blog! https://jaredallard.dev/fighting-back-against-ai-in-kubernetes-with-anubis/ works fine for me

It's probably because I'm on a released version, if I had to guess!

Reply:
Ah, guess I assumed anubis.techaro.lol was on a release version as well. Though dogfeeding makes sense